test/suites/drift/suite_test.go

```diff
@@ -65,8 +65,13 @@ var _ = Describe("Drift", Label("AWS"), func() {
 			"featureGates.driftEnabled": "true",
 		})
 		// choose an old static image
+		parameters, err := env.SSMAPI.GetParametersByPath(&ssm.GetParametersByPathInput{
+			Path:      awssdk.String("/aws/service/eks/optimized-ami/1.23/amazon-linux-2"),
+			Recursive: awssdk.Bool(true),
+		})
+		Expect(err).To(BeNil())
 		parameter, err := env.SSMAPI.GetParameter(&ssm.GetParameterInput{
-			Name: awssdk.String("/aws/service/eks/optimized-ami/1.23/amazon-linux-2/amazon-eks-node-1.23-v20221101/image_id"),
+			Name: awssdk.String(fmt.Sprintf("%s/image_id", *parameters.Parameters[len(parameters.Parameters)-2].Name)),
```

Comment:
Is this output guaranteed to be sorted?

Reply:
It has a specific hierarchy, according the documentation, but I will sort just in case